### PR TITLE
Increases tests coverage.

### PR DESCRIPTION
**Pull request by pwilczynskiclearcode**
* Separates __repr__ and __str__ of Executor
* Test cover of Exceptions

@fizyk Could you verify?

### CODE REVIEW

File: tests/executors/test_base.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+"""General mirakuru library tests."""
+
+
+def test_importing_mirakuru():
+    """Test if all most commonly used classes are imported by default."""
+    from mirakuru import *  # noqa
```
Comment (fizyk) [inline tests/executors/test_base.py]: from what I see that kind of import doesn't work for py3 ;) should be probably on module level

File: setup.py [hunk 2]
```diff
@@ -27,7 +27,8 @@
     package_version = re.compile(
         r".*__version__ = '(.*?)'", re.S).match(v_file.read()).group(1)
 
-tests_require = ['pytest', 'pytest-cov', 'mock']
+tests_require = (
+    'pytest', 'pytest-cov', 'mock', 'pytest-ipdb==0.1-prerelease2')
```
Comment (fizyk) [inline setup.py]: why do we need pytest-ipdb for tests?

Comment (pwilczynskiclearcode) [inline setup.py]: aaah, no, but it's helpful ;)

**Comment by fizyk** [comment]
:+1:

**Pull request merged**